docs/developer/app-store/apps/emails-and-messages/sendgrid.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Sendgrid provider
+sidebar_position: 2
+---
+
+# Sendgrid provider
+
+:::info
+
+Using this provider requires a Sendgrid account. You can create one [here](https://signup.sendgrid.com/).
+
+:::
+
+Sendgrid is a cloud-based email delivery and management platform that provides businesses with a reliable and scalable solution for sending transactional and marketing emails. It offers a set of tools and APIs that enable developers to easily integrate email functionality into their applications, websites, or software products.
```

Comment:
**polish**: In technical documentation, I think there is no need to emphasize the qualities of the software we are integrating with. Just lay down the dry facts.

```suggestion
Sendgrid is a cloud-based email delivery and management platform that provides businesses with a solution for sending transactional and marketing emails. It offers a set of tools and APIs that enable developers to integrate email functionality into their applications, websites, or software products.
```